docs/demo-vectortiles.html

```diff
@@ -7,21 +7,13 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 
 	<link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.3/dist/leaflet.css" />
-	<script src="https://unpkg.com/leaflet@1.0.3/dist/leaflet.js"></script>
-	<script src="https://unpkg.com/leaflet.vectorgrid@1.2.0"></script>
-<!-- 	<script src="../dist/Leaflet.VectorGrid.bundled.js"></script> -->
+	
 </head>
 <body style='margin:0'>
 	<div id="map" style="width: 100vw; height: 100vh; background: white"></div>
-
-
-<!--	<script type="text/javascript" src="../vendor/pbf-dev.js"></script>
-	<script type="text/javascript" src="../vendor/vector-tile-dev.js"></script>
-
-	<script type="text/javascript" src="../src/Leaflet.Renderer.SVG.Tile.js"></script>
-	<script type="text/javascript" src="../src/Leaflet.Renderer.Canvas.Tile.js"></script>
-	<script type="text/javascript" src="../src/Leaflet.VectorGrid.js"></script>
-	<script type="text/javascript" src="../src/Leaflet.VectorGrid.Protobuf.js"></script>-->
+	<script type="text/javascript" src="https://unpkg.com/leaflet@1.0.3/dist/leaflet.js"></script>
+	<!-- <script src="https://unpkg.com/leaflet.vectorgrid@1.2.0"></script> -->
+	<script type="text/javascript"  src="./../dist/Leaflet.VectorGrid.bundled.js"></script>

```

Comment:
Please note that this will fail "in production", i.e. when merged into the `master` branch and when a new build is released.